doc/en/apiref.rst

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
oh, since he changed each line wrt the extension its noticed as completely new file. git would only be able to get it right if he used separate commits for  rename and change